h_periodic/h_beat.py

```diff
@@ -55,8 +55,14 @@
         "fill-annotation-slim": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_annotation_slim",
-            "schedule": crontab(hour="9-13", minute="*/4"),
-            "kwargs": {"batch_size": 1000},
+            "schedule": crontab(hour="8", minute="*/3"),
+            "kwargs": {"batch_size": 500},
```

Comment:
I've seen the first(s) few batches of the day take longer. After that it seems to stabilize.

Run the first batches with a  smaller size.